lib/src/functions_client.dart

```diff
@@ -52,7 +52,7 @@ class FunctionsClient {
     final dynamic data;
     if (responseType == ResponseType.json) {
       final resBody = response.body;
-      data = resBody.isEmpty ? resBody : await compute(json.decode, resBody);
+      data = resBody.isEmpty ? resBody : await _isolate.decode(resBody);
```

Comment:
In postgrest, I added a check to only use the isolate for huge payloads https://github.com/supabase/postgrest-dart/blob/main/lib/src/postgrest_builder.dart#L215 I would add that here as well.

Reply:
Yeah, I was doing some testing, and I didn't really see any performance drop for small json strings either, so I was thinking of removing the check for whether the string is small or not and just pass all the string through the isolate methods. I think it's pretty safe to do so now that we are reusing the isolates.

Reply:
Okay 👍